packages/dds/sequence/src/defaultMap.ts

```diff
@@ -76,7 +76,7 @@ export interface IMapValueTypeOperation {
 	 * Value of the operation, specific to the value type.
 	 * @alpha
 	 */
-	value: IValueTypeOperationValue;
+	value: IValueTypeOperationValue | IIntervalDeltaOp;
```

Comment:
i don't think this is correct. IIntervalDeltaOp includes typings types that will never actually be on ops sent over the wire. it might make sense to pull the typing changes into another review.